src/cpp/session/SessionClientEventQueue.cpp

```diff
@@ -71,7 +73,14 @@ bool ClientEventQueue::setActiveConsole(const std::string& console)
 }
 
 void ClientEventQueue::add(const ClientEvent& event)
-{ 
+{
+   if (http_methods::connectionDebugEnabled())
```

Comment:
Consider adding a pref for this so we can all benefit!

Reply:
I added a preference so we can turn this on and off without restarting.